cms/djangoapps/contentstore/views/entrance_exam.py

```diff
@@ -128,6 +129,14 @@ def _create_entrance_exam(request, course_key, entrance_exam_minimum_score_pct=N
     }
     CourseMetadata.update_from_dict(metadata, course, request.user)
 
+    # Create the entrance exam section item.
+    create_xblock(
+        parent_locator=unicode(created_block.location),
+        user=request.user,
+        category='sequential',
+        display_name=_('Entrance Exam - Subsection')
```

Comment:
@asadiqbal08 - Apologies for the delay here, I thought the title above came out of doc review, and @srpearce I interpreted your comment above as "Subsection should be included in the display name."

My suggestion for this string would be to use "Exam"

Thanks @cahrens for flagging this. I had misread the earlier comments.